debian-config/packages.d/_ALWAYS.customise.add/etc/systemd/network/85-wlan-mesh.link

```diff
@@ -0,0 +1,14 @@
+# make the mesh-network devices /not/ match the default wlan hostap
+# By default, all wlan interfaces will be access points
+[Match]
+Type=wlan
+Driver=ath9k_htc rt2800usb rtl8192cu
+
+[Link]
+NamePolicy=kernel database
+MACAddressPolicy=persistent
+
+[mesh]
+country_code=CA
+ssid=tomesh
+freq=2412 HT40+
```

Comment:
Another thing to consider is having different files for different wifi dongles - as we know that there are some that support 5Ghz ranges, so they could have different config files to provide different "[mesh]" section values, and thus different frequencies.

Also, you are totally nuts for thinking that HT40 is usable in the 2Ghz bands

Reply:
But we want different hardware to talk to each other. If it's a known hardware node we should provide this file via conf.d. The default should optimize for letting most devices talk to each other.